_data/simple-icons.json

```diff
@@ -694,6 +694,11 @@
             "title": "Elastic Stack",
             "hex": "005571",
             "source": "https://www.elastic.co/brand"
+        },
+		{
```

Comment:
Can you fix the indentation here? Currently it seems to be to <kbd>tab</kbd>s whereas it should be spaces

---

<sup>Linting for this is being implemented in #1349</sup>